tests/unit/test_dto/test_factory/test_dataclass_dto.py

```diff
@@ -25,7 +25,7 @@ def fx_dto_type() -> type[DataclassDTO[Model]]:
     return DataclassDTO[Model]
 
 
-@pytest.mark.skipif(sys.version_info > (3, 8), reason="generic builtin collection")
+@pytest.mark.skipif(sys.version_info >= (3, 9), reason="generic builtin collection")
```

Comment:
`> (3, 8)` means we skip this test for any py version gt 3.8.0, which is basically all of them.